lib/web/machineid.go

```diff
@@ -28,6 +28,12 @@ import (
 	"github.com/gravitational/teleport/lib/reversetunnelclient"
 )
 
+const (
+	// webUIFlowBotGitHubActions is the key of the webUIFlowLabelKey
+	// added to a resource created via the Bot GitHub Actions web UI flow.
+	webUIFlowBotGitHubActions = "github-actions"
```

Comment:
Do we want to identify that this is the SSH specific flow ? e.g `github-actions-ssh` ?